python/src/aiconfig/editor/client/src/components/GlobalParametersContainer.tsx

```diff
@@ -30,7 +30,7 @@ export default memo(function GlobalParametersContainer({
       >
         <Accordion.Item value="parameters">
           <Accordion.Control>
-            <Text color="blue">Global Variables (Parameters) {"{}"}</Text>
+            <Text color="blue">Global Parameters</Text>
```

Comment:
Do we want to keep the {"{}"} here?

Reply:
hmmm, good question. We can keep it for now. Let me add that change.